frameworks/kafka/src/main/dist/server.properties.mustache

```diff
@@ -66,6 +69,18 @@ port={{KAFKA_BROKER_PORT_TLS}}
 {{/KAFKA_ALLOW_PLAINTEXT}}
 {{/KAFKA_ENABLE_TLS}}
 
+{{#SECURITY_KERBEROS_ENABLED}}
```

Comment:
This is obviously fine for MVP, but we will need to support TLS + SASL (e.g. SSL_SASL). 

And I actually remain unconvinced we should support anything but SSL_SASL for Kerberos.